SchulichEpsilonHermes/SchulichEpsilonHermes.cpp

```diff
@@ -0,0 +1,22 @@
+#include "InfrastructureLayer/InfrastructureContainer.h"
```

Comment:
I don't see the old files removed in this PR, they should be removed in this PR.